client/lib/pages/onboarding/onboarding_page.dart

```diff
@@ -118,11 +118,12 @@ class _OnboardingPageState extends State<OnboardingPage> {
       _showCountryListPage = false;
     });
     await _toNextPage();
+    var fcmToken = await UserPreferences().getFirebaseToken();
     try {
-      await widget.service
-          .putLocation(isoCountryCode: _selectedCountry.alpha2Code);
+      await widget.service.putClientSettings(
+          token: fcmToken, isoCountryCode: _selectedCountry.alpha2Code);
     } catch (error) {
-      print('Error sending location to API: $error');
+      print('Error sending client settings to API: $error');
```

Comment:
Ideally this should log an error in Analytics but lower priority for now.

```
widget.analytics.logEvent(....)
```

Reply:
Good idea. Maybe in who_service?
 widget.analytics.logEvent(
                        name: 'WhoServicePutClientSettings',
                        parameters: {'result': response.statusCode});

More important however is having the client know it should retry at some point. When should that be done? A flag could be saved in the prefs need_server_update, then at some time (e.g. the next launch it would know that it needs to try again.). Heck, that could reduce the 'race condition' by doing that as some sort of delayed event on the client. (E.g. save prefs; trigger async server update) But that seems like it should be a future change.